help.html

```diff
@@ -364,6 +425,23 @@ <h2 id="snippets">Snippets</h2>
         alt="http://goo.gl/bT8a8M"/></p>
     </div>
 
+    <h2 id="WebUSB">WebUSB</h2>
+    
+    <p><a herf="https://developers.google.com/web/updates/2016/03/access-usb-devices-on-the-web">WebUSB</a>
+    provides a simple way to connect to and control USB devices from the web. At present it is only available in
+    <a href="https://caniuse.com/#feat=webusb"> browsers that are based on Google Chrome</a>.</p>
+
+    <p>In the Python Editor, webUSB means that we can connect to the micro:bit, flash a .Hex file 
+    to it and communicate with it using the <a href="#REPL">REPL</a>.</p>
+
+    <p>If you see the message 'no compatible devices found' when you try to connect, 
+    you may need to <a href="https://microbit.org/guide/firmware">update the micro:bit firmware.</a></p>
+
```

Comment:
Can we add steps (or a gif) for "flash" and move the REPL section here?

Reply:
The reasoning behind this is to have all the WebUSB stuff together, so that when somebody in Internet Explorer clicks on "connect", get a warning about the browser and a link to the section, then that person can easily see all the WebUSB features that would be available in Chrome based browsers.

Reply:
done in https://github.com/bbcmicrobit/PythonEditor/pull/199/commits/d78d2bdff9d61c165df613e35fcf18347e544078